uninstall/uninstall_linux.go

```diff
@@ -0,0 +1,75 @@
+package uninstall
+
+import (
+	"time"
+	"io/ioutil"
+	"os/exec"
+	"github.com/pufferpanel/pufferd/logging"
+	"syscall"
+)
+
+
+func StartProcess(){
+	killDaemon()
+	deleteFiles()
+	deleteUser()
+}
+
+func killDaemon(){
+	exec.Command("systemctl", "pufferd", "stop").Run()
+	logging.Info("Attempting to kill all pufferd process...")
+	time.Sleep(time.Second * 5)//Giving 5 seconds to kill "correctly" all process

```

Comment:
Yeah, but what we want is to uninstall, so is it so important to kill pufferd in "the right way"?